### ISSUE TITLE
[generic/ubuntu2004]: Hyper-V cannot determine VM's IP address

### ISSUE THREAD

**Issue by pinger** [lavabit/robox#221]
Hi all,
It seems hyper-v box doesn't work. The VM does not show IP because some Hyper-V specific packets are missing.

generic/ubuntu2004 v3.5.2
host: Windows 11

Please add **linux-azure** package into the box.

![image](https://user-images.githubusercontent.com/1027141/141443143-cdeca004-7cda-48f4-bd58-30c72ddef905.png)

**Comment by ladar** [comment]
@pinger hi, I don't think it's the azure package that's needed. Recently there was an issue causing the packer builds to fail, so I had to remove the some of the packages from the install config. Specifically the relevant lines are:

```ubuntu1604: linux-tools-$(uname -r) linux-cloud-tools-$(uname -r) linux-cloud-tools-common linux-cloud-tools-generic linux-cloud-tools-virtual
ubuntu1610: linux-tools-$(uname -r) linux-cloud-tools-$(uname -r) linux-cloud-tools-common linux-cloud-tools-generic linux-cloud-tools-virtual
ubuntu1704: linux-tools-$(uname -r) linux-cloud-tools-$(uname -r) linux-cloud-tools-common linux-cloud-tools-generic linux-cloud-tools-virtual
ubuntu1710: linux-tools-$(uname -r) linux-cloud-tools-$(uname -r) linux-cloud-tools-common linux-cloud-tools-generic linux-cloud-tools-virtual
ubuntu1804: linux-tools-$(uname -r) linux-cloud-tools-$(uname -r) linux-cloud-tools-common linux-cloud-tools-generic linux-cloud-tools-virtual
ubuntu1810: linux-tools-$(uname -r) linux-cloud-tools-$(uname -r) linux-cloud-tools-common linux-cloud-tools-generic linux-cloud-tools-virtual
ubuntu1904: linux-tools-$(uname -r) linux-cloud-tools-$(uname -r)
ubuntu1910: linux-tools-$(uname -r) linux-cloud-tools-$(uname -r)
ubuntu2004: linux-tools-$(uname -r) linux-cloud-tools-$(uname -r)
```

I removed linux-cloud-tools-common linux-cloud-tools-generic linux-cloud-tools-virtual from the 19.04 to 20.04 configs. 20.10 and above use a different automated install process. 

That said, I though the linux-cloud-tools package was providing the Hyper-V modules. If it wasn't then packer wouldn't be able to figure out the IP address either. My guess is that kernel updates aren't tracking this because of the removed packages. I'll try adding them back and see if it works.

**Comment by ladar** [comment]
Seems to be working again. So it should be fixed in the 3.5.4  images... which are coming soon.
